docs/installation/_index.md

```diff
@@ -4,35 +4,217 @@ linkTitle: "Installation Guide"
+
+<br>
+
+```bash
+curl http://packages.viam.com/apps/viam-server/viam-server-stable-x86_64.AppImage -o viam-server &&
+  chmod 755 viam-server && sudo ./viam-server --aix-install
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you do not want to run `viam-server` as a service, you can also run it manually as a binary.
+
+</li>
+</ol>
+
+3. **Connect and configure.** Go to the **SETUP** page on the Viam app and wait for confirmation that your robot has successfully connected.
+
+{{% /tab %}}
+{{% tab name="macOS"%}}
+
+<br>
+
+`viam-server` is available for macOS users through [Homebrew](https://docs.brew.sh/Installation).
+
+1. **Install `viam-server` on your Mac**
+
+   ```bash
+   brew tap viamrobotics/brews && brew install viam-server
+   ```
+
+2. **Download the Viam app config to your Mac.** `viam-server` uses this config file to connect to app.viam.com.
+   This connection allows the robot to pull its full configuration information and allows you to monitor and control your robot from the Viam app.
+   Download your robot's config file from the **SETUP** tab of your robot on the Viam app.
+
+3. **Start `viam-server` on your Mac.** Run viam-server locally on your Mac with the config you just downloaded.
+   Replace `<YOUR_ROBOT_NAME>` with the name of your robot from the Viam app.
+
+   ```bash
+   viam-server -config ~/Downloads/viam-<YOUR_ROBOT_NAME>-main.json
+   ```
+
+4. **Connect and configure.** Go to the **SETUP** page on the Viam app and wait for confirmation that your robot has successfully connected.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Alternative ways of running `viam-server` on Linux
+
+To run `viam-server`, you have two options:
+
+#### Run Directly from the Command Line
+
+To run `viam-server` directly from the command line, you can use the following command, replacing "myconfig" with the name of your configuration file.
+`sudo` is necessary on some devices to access hardware.
+
+```bash
+sudo ./viam-server -config myconfig.json
+```
+
+#### Install as a System Service
+
+A system service starts `viam-server` every time you boot your device.
+
+For this setup your configuration file must be at <file>/etc/viam.json</file>.
+
+The following command creates a systemd service file at <file>/etc/systemd/system/viam-server.service</file> and sets it to start on boot, using a config placed at <file>/etc/viam.json</file>.
+It will also move the actual binary (AppImage) to <file>/usr/local/bin/viam-server</file> (regardless of the previous filename).
+
+```bash
+sudo ./viam-server --aix-install
+```
+
+Start the service by running:
+
+```bash
+sudo systemctl start viam-server
+```
+
+The service is an AppImage and will check for updates and self-update automatically each time the service is started.
+Self-updates can take a couple of minutes, so the service may sometimes take a moment to start while this runs.
+You can disable this by commenting out the ExecPre line (the one with --aix-update on it) in the service file.
```

Comment:
please do not review this part - We'll assess whether to merge this with the manage or update section in a separate PR.